src/common/Messages.h

```diff
@@ -21,7 +21,13 @@ enum MessageOffset : unsigned int
 {
     TYPE = 0,
     INST = 1,
-    BODY = 2
+    BODY = 2,
```

Comment:
Any reason for the extra comma? There are other with comma and some without comma. We fix to one convention and  be consistent.